example-log-bucket/s3.tf

```diff
@@ -0,0 +1,175 @@
+/*
+ * Make sure that you use the latest version of the module by changing the
+ * `ref=` value in the `source` attribute to the latest version listed on the
+ * releases page of this repository.
+ *
+ */
+module "example_team_s3_bucket" {
+  #source = "github.com/ministryofjustice/cloud-platform-terraform-s3-bucket?ref=4.1"
```

Comment:
I think we need to leave this as is, possibly changing the version number to 4.2

Users will copy and paste this code for themselves, so the `../` path won't work for them, and it's one less thing they'll have to change.